src/trello_tasks_backend.js

```diff
@@ -0,0 +1,39 @@
+const superagent = require('superagent')
+const TasksBackend = require('./tasks_backend')
+
+const TRELLO_API = 'https://api.trello.com/1'
```

Comment:
Can this be made a parameter that's passed to `init`?

Reply:
Done! I'll request another review to cover that change.